crates/isograph_schema/src/unvalidated_schema.rs

```diff
@@ -47,6 +53,11 @@ pub type UnvalidatedVariableDefinition = VariableDefinition<
     <UnvalidatedSchemaState as SchemaValidationState>::VariableDefinitionInnerType,
 >;
 
+pub type UnvalidatedClientPointer = ClientPointer<
+    <UnvalidatedSchemaState as SchemaValidationState>::ClientPointerSelectionScalarFieldAssociatedData,
+    <UnvalidatedSchemaState as SchemaValidationState>::ClientPointerSelectionLinkedFieldAssociatedData,    <UnvalidatedSchemaState as SchemaValidationState>::VariableDefinitionInnerType,
```

Comment:
spacing looks weird. Sometimes when the line is too long, `cargo fmt` fails to pick up on it. Can you fix this manually?